packages/core/src/index.ts

```diff
@@ -68,3 +68,15 @@ export * from "./extensions/UniqueID/UniqueID.js";
 export * from "./api/exporters/markdown/markdownExporter.js";
 export * from "./api/parsers/html/parseHTML.js";
 export * from "./api/parsers/markdown/parseMarkdown.js";
+
+export * from "./extensions/Comments/types.js";
+
+// TODO: how to organize this? The number of exports is growing
+// - we could namespaces things (i.e.: comments.yjs.YjsThreadStore)
+// - split to different package (@blocknote/comments)
+// - sub imports ( @blocknote/core/comments)
```

Comment:
Big fan of sub-exports so you can keep bundle size to a minimum. Will need some tooling to split up the bundling